### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Chintesh**
Documentation updated.

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,12 +103,12 @@ We typically update at minimum the following environment variables in the `.env`
 | Variable | Description |
 | :--- | :--- |
 | `API_ADMIN_EMAIL` | The email associated with the local admin account. Default is `admin@tide.local`. _Note: unless you setup the Gmail SMTP plugin, any generated emails will not get sent by WordPress._ |
-| `API_ADMIN_PASSWORD` | The password associated with the local admin account. Default is `wordpress`. |
+| `API_ADMIN_PASSWORD` | The password associated with the local admin account. Default is `WordPress`. |
```
Comment (jeffpaul) [inline README.md:106]: This is an incorrect change as the password is case sensitive, please revert.

File: README.md [hunk 2]
```diff
@@ -103,12 +103,12 @@ We typically update at minimum the following environment variables in the `.env`
...
 | `API_KEY` | The API key used both locally and on GCP to authenticate the `audit-server` user. Default is `uRhZgeYt$v5u0vR5fpcDSWcZU`. |
 | `API_SECRET` | The API secret used both locally and on GCP to authenticate the `audit-server` user. Default is `rVvUWQlPQr8trSEd2qdwmE4Eiua$MjLX`. |
 
-To make local development simple we have added default values for the `API_KEY` and `API_SECRET` associated with the `audit-server` user, which will automatically update the user meta values when `make api.setup` is ran. However, you are free to change these values and we encourage you to, especially if you plan on deploying to the cloud — in that case you should overwrite these values in the `.env.gcp` file.
+To make local development simple we have added default values for the `API_KEY` and `API_SECRET` associated with the `audit-server` user, which will automatically update the user meta values when `making api.setup` is ran. However, you are free to change these values and we encourage you to, especially if you plan on deploying to the cloud — in that case you should overwrite these values in the `.env.gcp` file.
```
Comment (jeffpaul) [inline README.md:111]: This is an incorrect change as the command is `make` and not `making`, please revert.

**Review by jeffpaul — CHANGES_REQUESTED** [review]
See inline changes needed

**Comment by jeffpaul** [comment]
@Chintesh are you able to make the updates that I noted?

**Comment by valendesigns** [comment]
@Chintesh Thank you for bringing this to our attention. I've updated the typo in the `develop` branch. The other two changes were not correct. I want you to know that we appreciate your contribution, but a simple text change will not get anyone contributor status. I have to manually change several files to add a contributor and then deploy everything to App Engine. It's a lot of effort for such a small thing. If you want to get added to the list it will have to be an important text change or actual code contribution. Thank you though, and I hope you still want to contribute and help the project.

**Pull request closed without merging**